test/java/com/arpnetworking/testing/SerializationTestUtils.java

```diff
@@ -29,7 +37,40 @@
 public final class SerializationTestUtils {
 
     // TODO(spencerpearson): allow the ObjectMapper to get passed in, if ever necessary
-    private static final ObjectMapper OBJECT_MAPPER = ObjectMapperFactory.getInstance();
+    // IMPORTANT: The configuration here must match that in MainModule for testing REST APIs
```

Comment:
Doesn't have to be part of this diff, but: would it be feasible to refactor the ObjectMapper-instantiation out of MainModule into something that the test could import?

Reply:
I give it a shot on the next one.